server/routes/createGoal/createGoalController.ts

```diff
@@ -118,6 +118,14 @@ export default class CreateGoalController {
     const { prisonerSummary } = req.session
     const { prisonId } = prisonerSummary
 
+    const reviewGoalForm = { ...req.body }
```

Comment:
This is the `submitReviewGoal` controller method
It now looks at the request body (`ReviewGoalForm`) and the `action` field within it - if it is `add-another-goal` then we reset the `NewGoal` session variable (singular, not the plural array), and redirect back to the start of the journey (ie. the 1st Create Goal screen)